Makefile

```diff
@@ -3,6 +3,9 @@ SHELL := /bin/bash # Use bash syntax
 # Set up variables
 GO111MODULE=on
 
+ELASTICACHE_API_PATH="$(shell echo $(GOPATH))/pkg/mod/github.com/aws/aws-sdk-go@v1.37.4/service/elasticache/elasticacheiface"
```

Comment:
approving this for now, but you'll likely want to do a little `awk` or `grep`-fu to replace the `1.37.4` hard-coding above by looking in the go.mod file :)